### ISSUE TITLE
Feature: Config option to disable auto generate routing

### ISSUE THREAD

**Issue by gbourant** [quarkiverse/quarkus-renarde#230]
At the moment Renarde (`RenardeProcessor#collectControllers`) generates routes for all non private methods of a given Controller.

The thing is, you could expose sensitive data, if you forget to define the method as private.

**Comment by FroMage** [comment]
If you make the method `public` then it's exposed, that's the contract. There's no point in making public methods in a controller other than to expose them, so I'm not sure this is an issue.

**Comment by mattgreen** [comment]
FWIW, I bumped into this at one point: I wanted to implement `RenardeOidcHandler` in a `Controller` that had other auth-related concerns, as `RenardeOidcHandler` behaves similarly to a Controller sans routing.

Would have been nice to be able to hide the methods with an annotation. Maybe wait and see if other people have other use cases that are blocked by the inability to hide public methods from routing?

**Comment by FroMage** [comment]
Yeah, `RenardeOidcHandler` is special, and not something that can be implemented in a `Controller`. Perhaps I should make it an explicit error if you try that?

**Comment by mattgreen** [comment]
Seems reasonable! A mention in the docs would also be sufficient.
